worldedit-core/src/main/java/com/sk89q/worldedit/command/ToolUtilCommands.java

```diff
@@ -54,15 +55,15 @@ public void togglePickaxe(Player player, LocalSession session,
                                   Boolean superPickaxe) {
         boolean hasSuperPickAxe = session.hasSuperPickAxe();
         if (superPickaxe != null && superPickaxe == hasSuperPickAxe) {
-            player.printError("Super pickaxe already " + (superPickaxe ? "enabled" : "disabled") + ".");
+            player.printError(TranslatableComponent.of(superPickaxe ? "worldedit.tool.superpickaxe.enabled.already" : "worldedit.tool.superpickaxe.disabled.already"));
```

Comment:
I feel like we could benefit from having a top-level `worldedit.feature.already.enabled`/`.disabled`, that takes an argument like `worldedit.tool.superpickaxe`, with English TR like `%s already enabled`. That way we don't have a separate key for each individual tool's already enabled/disabled. Could be applicable to the plain `enabled`/`disabled` messages as well.

Reply:
So I had a look at this, and tbh I don't think there are enough instances to really warrant it